src/sources.js

```diff
@@ -323,6 +325,16 @@ class S3Source extends Source {
     }
     return object.Body;
   }
+  parseOverviewMarkdown(overviewMarkdown) {
+    const frontMatter = yamlFront.loadFront(overviewMarkdown);
+    if (!frontMatter.title || !frontMatter.byline) {
```

Comment:
It's probably ok to make `title` compulsory but `byline` should be optional. If the markdown exists but `byline` is missing then we shouldn't display one.